packages/cx-elements/doc/Design/CanvasxTokens/TokenDomains.md

```diff
@@ -0,0 +1,153 @@
+
+## Allowed Domains
+
+The set of allowed domains is defined by CanvasX as follows.  For most domains, the
+value is a "extended" design object (details below).  Three domains ('Meta','Compose'
+and 'Flow') are treated differently as described below.
+
+- `Core`: Essential behavior or styling added by this token which are
+  very unlikely to be overridden; eg, accessibility tokens.
+- `Analytics`: Behavior or props related to analytics; eg. pushing events to a
+  data layer.
+- `SEO`: Behavior or props related to search engine optimization, eg adding
+  [schema.org](https://schema.org/) markup.
+- `Components`: When the design elements of a complex component are themselves
+  complex components, it is generally best practice to define tokens which apply
+  to the sub-components as a whole, and apply them in the `Components` domain
+  of the enclosing component.
+- `Layout`: Tokens which define the visual structure of a component and are thus
+  unlikely to be overridden; eg those which define the orientation of a card.
+- `Spacing`: Tokens which sit somewhere between `Theme` and `Layout`; eg
+  padding, margin, line-spacing, etc.
+- `Theme`: Tokens which apply styling which is very likely to be overridden;
+  eg colors, typography, etc.
+- `Editors`: Tokens which define how a component's data are edited.
+- `Content`: Tokens which provide default content or other fixed props. Any
+  hardcoded, translatable strings belong in this domain.
+- `Behavior`: Tokens which define or add behaviors to a component; eg the
+  expanding and contracting of an accordion.
+- `Schema`: Tokens which define how a component's data are organized; 
+  eg, node keys.
+
+### Special Domains
+
+The following special domains have values which are not extended domain
+objects:
+
+- `Compose`: This special domain allows specifying other named tokens which
+  should be composed with this one. See
+  [Composition and Extension](../CanvasxTokens#composition-and-extension) for
+  more details. Example:
+  ```ts
+  const Sticky = asHeaderToken({
+    ...Base,
+    Compose: {
+      // Apply the WithSticky token to the component as a whole.
+      WithSticky,
+    },
+  })
+  ```
+- `Flow`: By default, the tokens HOC's for each domain are composed using
+  the Bodiless `asToken` utility.  This behavior can be overridden by specifying
+  a different composer here.  This is useful for wrapping the components in a
+  condition (using `flowIf`).
+  ```ts
+  const WithBlueIfActive = asTokenSpec()({
+    Core: {
+      _: 'text-blue',
+    },
+    Flow: flowIf(({ isActive }) => Boolean(isActive)),
+  }),
+  ```
+- `Meta`: This Domain takes a token metadata object which should be attached to
+  the token (and to any component to which the token is applied).
+  ```ts
+  const WithBlueBackground = asTokenSpec()({
+    Core: {
+      _: 'bg-blue',
+    },
+    Meta: asToken.meta.term('Color')('Blue'),
+  });
+  ```
+
+## Order ot domains
```

Comment:
```suggestion
## Order of domains
```

Reply:
@hvanyo ,

I'll merge the PR and address it directly in my branch. Thanks!